.github/workflows/test_benchmark_collection_models.yml

```diff
@@ -60,6 +60,12 @@ jobs:
         git clone --depth 1 https://github.com/benchmarking-initiative/Benchmark-Models-PEtab.git \
           && export BENCHMARK_COLLECTION="$(pwd)/Benchmark-Models-PEtab/Benchmark-Models/" \
           && AMICI_PARALLEL_COMPILE=2 tests/benchmark-models/test_benchmark_collection.sh
+          
+    # run gradient checks
+    - name: Run Gradient Checks
+      run: |
+        pip install git+https://github.com/ICB-DCM/fiddy.git \
```

Comment:
Does that reuse the imported models from the test above, or does it re-import everything?